app/models/events/comment_replied_to.rb

```diff
@@ -3,10 +3,14 @@ class Events::CommentRepliedTo < Event
 
   def self.publish!(reply)
     return unless reply.parent.present?
-    create!(kind: 'comment_replied_to',
-            eventable: reply,
-            user: reply.author,
-            created_at: reply.created_at)
+    event = create!(kind: 'comment_replied_to',
```

Comment:
Since you're aping other methods in the controller, this should stay as is, but FYI a more better way, with tap:

```
create!(kind: 'comment_replied_to', 
        eventable: reply,
        user: reply.parent.author, 
        created_at: reply.created_at).tap do |event|
  ThreadMailer.delay.comment_replied_to reply.parent.author, event
end
```